moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -34,22 +34,21 @@
  *********************************************************************/
 
 /*
- * IKFast Plugin Template for moveit
+ * IKFast Kinematics Solver Plugin for MoveIt! wrapping an ikfast solver from OpenRAVE.
  *
- * AUTO-GENERATED by create_ikfast_moveit_plugin.py in arm_kinematics_tools
- * You should run create_ikfast_moveit_plugin.py to generate your own plugin.
- *
- * Creates a kinematics plugin using the output of IKFast from OpenRAVE.
- * This plugin and the move_group node can be used as a general
- * kinematics service, from within the MoveIt! planning environment, or in
- * your own ROS node.
+ * AUTO-GENERATED by create_ikfast_moveit_plugin.py in moveit_kinematics package.
  *
  */
 
 #include <ros/ros.h>
 #include <moveit/kinematics_base/kinematics_base.h>
-#include <urdf/model.h>
+#include <moveit/robot_state/robot_state.h>
+#include <Eigen/Geometry>
 #include <tf2_kdl/tf2_kdl.h>
+#include <tf2_eigen/tf2_eigen.h>
+#include <eigen_conversions/eigen_kdl.h>
+
+using namespace moveit::core;
```

Comment:
this is very bad practice, at minimum please provide an alias e.g. ``mc::``

Reply:
It's a no-go for a header file, but absolutely fine for a cpp, isn't it?

Reply:
ah ok, fine.